js/scorm/wrapper.js

```diff
@@ -595,12 +595,16 @@ class ScormWrapper {
       // range split into negative/positive ranges (rather than minScore-maxScore) depending on score
       const range = (score < 0) ? Math.abs(minScore) : maxScore;
       // `scaled` converted to -1-1 range to indicate negative/positive weighting now that negative values can be assigned to questions
-      const scaledScore = score / range;
+      const scaledScore = (score === 0 && range === 0)
```

Comment:
```suggestion
      const scaledScore = (range === 0)
```
![image](https://github.com/user-attachments/assets/5cb6f874-d7d4-4d1a-8c22-f4f50964c567)